components/playlist/browser/resources/components/contextualMenu.tsx

```diff
@@ -23,7 +23,7 @@ interface MenuProps {
   onDismissMenu?: () => void
 }
 
-const StyledRow = styled('leo-menu-item')`
```

Comment:
Interestingly, when <leo-menu-item> is slotted, it seems the tag loses the styles🤔

Reply:
Probably some weird slotting interaction - I think the `styled-components` adds the styles to the root document, whereas the element is being rendered in the ShadowDOM (which doesn't see the `styled-components` stylesheets).